submissions/MutmainahOdunewu/html-forms/contact.html

```diff
@@ -30,14 +30,18 @@ <h2>Email Address <br>support@sandalsbymo.com </h2>
              </ul>
          </div>
          <div class="side">
-            <form>
+            <form id = "validateForm">
                 <img src="../HTML/images/closed.png" alt="closed">
                 <p>Or better still send us a message</p>
-                <input type="text" name="fullname" id="fullname" placeholder="Fullname"/><br><br>
-                <input type="text" name="email"  placeholder="Email Address"/><br><br>
+                <input type="text" name="fullname" id="fullName" placeholder="Fullname"/><br><br>
```

Comment:
Using `nextElementSiblings` about 3 times is bad. It means your code is not properly structured. Kindly remove all the `br` tags and use CSS for your styling instead